test/introspection/ERC165.test.js

```diff
@@ -9,6 +9,12 @@ contract('ERC165', function (accounts) {
     this.mock = await ERC165Mock.new();
   });
 
+  it('register interface', async function () {
```

Comment:
I think this test should be removed since it pertains `ERC165Storage` and not plain `ERC165`.